From 48957476c94fcd622e317ce551b3057e7ebd1a51 Mon Sep 17 00:00:00 2001
From: Wang Panzhenzhuan <randy.wang@rock-chips.com>
Date: Wed, 16 Oct 2019 17:47:35 +0800
Subject: [PATCH] arm64: configs: rockchip_defconfig: enable ov13850 & vcm149c
 & SGM3784

OV13850 Camera Module used by rk3399 & rk3399pro excavator board

Change-Id: Ieb06cc519609392475db1d4f0103a89d22110fd4
Signed-off-by: Wang Panzhenzhuan <randy.wang@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 6b78717a656b..b80461e38812 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -533,6 +533,9 @@ CONFIG_VIDEO_ROCKCHIP_ISP1=y
 # CONFIG_MEDIA_SUBDRV_AUTOSELECT is not set
 CONFIG_VIDEO_GC0312=y
 CONFIG_VIDEO_GC2145=y
+CONFIG_VIDEO_OV13850=y
+CONFIG_VIDEO_VM149C=y
+CONFIG_VIDEO_SGM3784=y
 CONFIG_CAMSYS_DRV=y
 CONFIG_FLASHLIGHT=y
 CONFIG_LEDS_RT8547=y
-- 
2.35.3

